apps/site/components/Common/AvatarGroup/index.tsx

```diff
@@ -1,25 +1,31 @@
 'use client';
 
 import classNames from 'classnames';
-import type { ComponentProps, FC } from 'react';
-import { useState, useMemo } from 'react';
+import type { FC } from 'react';
+import { useState, useMemo, Fragment } from 'react';
 
+import type { AvatarProps } from '@/components/Common/AvatarGroup/Avatar';
 import Avatar from '@/components/Common/AvatarGroup/Avatar';
 import avatarstyles from '@/components/Common/AvatarGroup/Avatar/index.module.css';
-import { getAcronymFromString } from '@/util/stringUtils';
+import AvatarOverlay from '@/components/Common/AvatarGroup/Overlay';
+import Tooltip from '@/components/Common/Tooltip';
 
 import styles from './index.module.css';
 
 type AvatarGroupProps = {
-  avatars: Array<Omit<ComponentProps<typeof Avatar>, 'fallback'>>;
+  avatars: Array<AvatarProps & { url?: string }>;
```

Comment:
Why is URL not part of AvatarProps?

Reply:
It was unnecessarily redefined, I removed it 👍